### DISCUSSION TITLE
Github pages stopped working suddenly

### DISCUSSION THREAD

**Discussion by ThePsychDr** [community/community#46186] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I hosted a github page for about a week and now when i try to access it i get this error
<img width="776" alt="CleanShot 2023-02-02 at 15 06 14@2x" src="https://user-images.githubusercontent.com/111861904/216438164-43550bd3-ab52-4196-9a90-0d03ca047b0f.png">

I have made no changes in between when i first published and now, I tried reposting the page and deleting the index as it points to nowhere. 
<img width="617" alt="CleanShot 2023-02-02 at 15 05 52@2x" src="https://user-images.githubusercontent.com/111861904/216438069-02648c30-1282-4767-a915-158edf0fa6a3.png">

**Comment by SilasRodrigues19** [comment, marked as answer]
Try rename your html file as `index.html` and deploy again

**Reply by ThePsychDr** [reply]
Has not worked.

**Reply by ThePsychDr** [reply]
Works now! https://thepsychdr.github.io/Resume/Index.html have to access it through index. Thank you.

**Reply by SilasRodrigues19** [reply]
Rename to `index.html` in lowercase to access by https://thepsychdr.github.io/Resume/
